corehq/apps/api/odata/utils.py

```diff
@@ -45,7 +45,7 @@ def get_properties_by_xmlns(domain, app_id, xmlns):
     ) or FormExportDataSchema.generate_schema_from_builds(domain, app_id, complete_xmlns)
     export_items = [
         item for item in form_export_schema.group_schemas[0].items
-        if isinstance(item, ExportItem)
+        if isinstance(item, ExportItem) and len(form_export_schema.group_schemas)
```

Comment:
I'm a bit confused how this would ever be false since you are iterating over an item of `form_export_schema.group_schema`? so wouldn't `form_export_schema.group_schemas` always have a length?

Reply:
whoops: 9b8c32b